tests/Nelmio/Alice/Fixtures/Parser/Methods/PhpTest.php

```diff
@@ -110,4 +110,31 @@ public function testIncludeFiles()
         ];
         $this->assertEquals($expectedData, $data);
     }
+
+    public function testIncludeFilesOrder()
+    {
+        $data = $this->parser->parse(__DIR__.'/../../../support/fixtures/include_order.php');
+
+        $expectedData = [
+            'Foo' =>
+                [
+                    'foo' =>
+                        [
+                            'id' => 1,
+                            'text' => '<word()>',
+                        ],
+                ],
+            'Bar' =>
+                [
+                    'bar' =>
+                        [
+                            'id' => 1,
+                            'text' => '<@foo->text>'
+                        ],
+                ],
+        ];
+
+        $this->assertEquals($expectedData, $data);
+        $this->assertTrue($expectedData === $data);
```

Comment:
Sure, this is exactly what we want to test.

The order returned by parse() must be exactly the same as in $expectedData, otherwise we have lost the "include" order as defined in the parsed fixture file.
Please refer to issue #314 for further details.

Now that this is clarified, what are we going to do ?
Do you think it would be better to use assertEquals() + assertTrue() or assertSame() only ? :)